terrawrap/utils/cli.py

```diff
@@ -72,6 +75,16 @@ def execute_command(
             if value is not None
         }
 
+    timestamp = int(time.time())
```

Comment:
looks like you changed it to float in the other pr so you should leave it as float here too